src/Features/Core/Portable/GenerateType/AbstractGenerateTypeService.cs

```diff
@@ -72,7 +73,18 @@ protected AbstractGenerateTypeService()
                 var state = State.Generate((TService)this, semanticDocument, node, cancellationToken);
                 if (state != null)
                 {
-                    return GetActions(semanticDocument, node, state, cancellationToken);
+                    var actions = GetActions(semanticDocument, node, state, cancellationToken).ToList();
```

Comment:
```ToList()``` always makes me cry a little. It seems kind of necessary for the code below though.